eng/Publishing.props

```diff
@@ -0,0 +1,14 @@
+<Project>
+  <PropertyGroup>
+    <ProducesDotNetReleaseShippingAssets>true</ProducesDotNetReleaseShippingAssets>
```

Comment:
This should be false.

Reply:
Or removed.

Reply:
Fixed with https://github.com/NuGet/NuGet.Client/pull/5673/commits/1e9379beb8e4fa6c66e92a155970f43858c7e184

Reply:
> Fixed with [1e9379b](https://github.com/NuGet/NuGet.Client/commit/1e9379beb8e4fa6c66e92a155970f43858c7e184)

@jeffkl if we could have one more CI run to verify this change, that would be great. After that passes, the PR can be merged.